src/components/skeletons/Shimmer/Shimmer.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+
+import './shimmer.scss'
+
+const Shimmer = () => {
```

Comment:
please use functions for components so React devTools print the component's name